pixeltable/env.py

```diff
@@ -265,6 +265,18 @@ def is_local(self) -> bool:
         assert self._db_url is not None  # is_local should be called only after db initialization
         return self._db_server is not None
 
+    def is_interactive(self) -> bool:
+        """Return True if running in an interactive environment."""
+        if getattr(builtins, '__IPYTHON__', False):
+            return True
+        # Python interactive shell
+        if hasattr(sys, 'ps1'):
+            return True
+        # for script execution, __main__ has __file__
```

Comment:
Are we sure we want to treat this as "interactive"? It's not so clear to me.

Reply:
Which part isn't clear? Is there a better way to check for this?

Reply:
Ah I think you misinterpreted my comment. I'm saying maybe we shouldn't be treating script execution as interactive.